docs/pages/includes/config/cloud.mdx

```diff
@@ -0,0 +1,100 @@
+## Configuration resource
```

Comment:
```suggestion
```
I think we should remove this heading to turn this paragraph into an intro for the whole Cloud config reference.

Reply:
I would generally agree, except that it provides symmetry with the OSS and Enterprise versions, which have **teleport.yaml** header.

Reply:
I went back and forth on this, and ended up thinking: Yes, but only if we also remove the **teleport.yaml** header on the OSS/Enterprise side.